src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
 spring.application.name=harmoniq
+main.url=https://harmoniq-1.onrender.com
```

Comment:
I think this should also be in `.env` because it would be localhost for local development

Reply:
Ok It's been fixed